cpp/src/ice2slice/Gen.cpp

```diff
@@ -202,18 +190,18 @@ namespace
         ostringstream os;
 
         static const char* builtinTable[] = {
-            "byte",                            // KindByte
-            "bool",                            // KindBool
-            "short",                           // KindShort
-            "int",                             // KindInt
-            "long",                            // KindLong
-            "float",                           // KindFloat
-            "double",                          // KindDouble
-            "string",                          // KindString
-            "???",                             // KindObject
-            "IceRpc.Slice.Ice.IceObjectProxy", // KindObjectProxy
-            "???",                             // KindLocalObject
-            "???"                              // KindValue
+            "byte",                             // KindByte
+            "bool",                             // KindBool
+            "short",                            // KindShort
+            "int",                              // KindInt
+            "long",                             // KindLong
+            "float",                            // KindFloat
+            "double",                           // KindDouble
+            "string",                           // KindString
+            "???",                              // KindObject
+            "IceRpc.Slice.Ice.IceObjectProxy?", // KindObjectProxy
+            "???",                              // KindLocalObject
```

Comment:
Something is wrong here, since we no longr have local object in Ice 3.8.

Reply:
The current list of built in types (in order) is:
```
KindByte,
KindBool,
KindShort,
KindInt,
KindLong,
KindFloat,
KindDouble,
KindString,
KindObject,
KindObjectProxy,
KindValue
```
So just removing the `KindLocalObject` entry is sufficient.